packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -146,6 +147,7 @@ const POISubmission = ({
                             residence_list={residence_list}
                             handleSubmit={handleViewComplete}
                             latest_status={identity_last_attempt}
+                            chosen_country={selected_country}
```

Comment:
can we name this props as `selected_country` to be in sync with the name everywhere?